.changeset/quick-pillows-jump.md

```diff
@@ -3,10 +3,10 @@
 "fluid-framework": minor
 "__section": tree
 ---
-New allowUnused utility function
+allowUnused utility function
```

Comment:
Tangent: is this really an API we want to expose out of the tree package? It's alpha, we can revert it later, but I'm very skeptical that this belongs in tree., especially since our own code doesn't use it outside the package.

Reply:
It's an alpha API so we can change it, and lack of such an API makes it impossible for some of our own packages to follow the documented API usages practices for working around TypeScript limitations. See ValidateRecursiveSchema, and https://github.com/microsoft/FluidFramework/pull/24058 for some examples of how updating our own code to follow our own docs fails to build without this function